bucket/featherwallet.json

```
@@ -0,0 +1,39 @@
+{
+    "version": "1.0.1",
+    "description": "A Monero SPV Wallet",
+    "homepage": "https://featherwallet.org/",
+    "license": {
+        "url": "https://git.featherwallet.org/feather/feather/raw/branch/master/LICENSE"
```

```suggestion
        "identifier": "BSD-3-Clause",
        "url": "https://git.featherwallet.org/feather/feather/raw/branch/master/LICENSE"
```
It is actually a [BSD 3-Clause license](https://opensource.org/licenses/BSD-3-Clause)

---

Also we don't need the URL as well

---

> Also we don't need the URL as well

I tend to keep the URL, because this saves our time to find the license if something happens in the future like:
(1) Someone make a complaint that license is not correct
e.g. `LICENSE` exists in the repo because some library has it, but the software applies another rule.
e.g. The author of the PR mis-read the license as some generic one (like MIT), and our maintainers didn't find out.
(2) License changed by the developer.

I don't think it should be enforced to keep the URL, though.